### PR TITLE
Register tr1xem.is-a.dev

### PR DESCRIPTION
**Pull request by tr1xem**
Register tr1xem.is-a.dev with TXT record pointing to 7bf3c01b64e04e87d52e935076aa7a.

### CODE REVIEW

File: domains/tr1xem.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "tr1xem",
+           "email": "kmtechnical2006@gmail.com",
+           "discord": "885063317079592961"
+        },
+    
+        "record": {
+            "TXT": "7bf3c01b64e04e87d52e935076aa7a"
```
Comment (DEV-DIBSTER) [inline domains/tr1xem.json:9]: TXT record alone doesn't do anything.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/tr1xem.json (hunk 1 above)
Comment (tr1xem) [reply domains/tr1xem.json:9]: Can you delete it?

File: domains/it.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+        "owner": {
+           "username": "tr1xem",
+           "email": "kmtechnical2006@gmail.com",
+           "discord": "885063317079592961"
+        },
+    
+        "record": {
+             "TXT": [
```
Comment (SX-9) [inline domains/it.json:9]: invalid txt record (txt can only be string) and txt alone isnt enough

**Review by satr14washere — CHANGES_REQUESTED** [review]
invalid format

**Comment by SX-9** [comment]
closing this as it is the same as #10877

**Pull request closed without merging**